pkg/plugins/policies/meshtcproute/plugin/v1alpha1/listeners_backendrefs.go

```diff
@@ -0,0 +1,56 @@
+package v1alpha1
+
+import (
+	core_mesh "github.com/kumahq/kuma/pkg/core/resources/apis/mesh"
+	core_xds "github.com/kumahq/kuma/pkg/core/xds"
+	api "github.com/kumahq/kuma/pkg/plugins/policies/meshtcproute/api/v1alpha1"
+)
+
+func matchingHTTPRuleExist(
```

Comment:
If we agree on previous comments, then this file is going to have only this function, so I'd move it to `listeners.go`

Reply:
As suggested, I used `core_xds.ComputeConf`, but I still left this separated in two distinct functions as it's more clean to me, and makes reading code easier (at least for me). As commented in regards of filterChain, I would prefer to leave it in separate file as well. Please reopen the conversation if you strongly disagree.